sim-cli/src/config.rs

```diff
@@ -0,0 +1,209 @@
+use std::{path::PathBuf, str::FromStr};
+
+use anyhow::Context;
+use log::LevelFilter;
+use serde::Deserialize;
+
+use crate::Cli;
+
+#[derive(Debug)]
+pub struct SimulationConfig {
```

Comment:
Can we derive serde's `Serialize` and `Deserialize` here and then provide a custom serialization for log level to save some code? Like we do for [lnd](https://github.com/bitcoin-dev-project/sim-ln/blob/main/sim-lib/src/lnd.rs#L30) paths.

Reply:
Done